react/features/browser-capabilities/capabilities.json

```diff
@@ -0,0 +1,71 @@
+{
+    "Safari": [
+        {"version": "6" },
+        {
+            "version": "10",
+            "capabilities": {
+                "audioIn": true,
+                "audioOut": true,
+                "video": true,
+                "screenSharing": true
+            }
+        },
+        { }
```

Comment:
Because there is no entry for Safari 11 does that mean it's considered unsupported? What does this empty object mean? Safari 11 at least supports audio only, no?

Reply:
Yes, it means it is not supported. I tested and audio only works. I got confused with the iframe use case. It should be fixed now.